docs/tutorial/part-one/index.md

```diff
@@ -324,22 +326,22 @@ export default () => (
 )
 ```
 
-After you save the file, you should see the contact page and be able to follow the link to the homepage.
+फ़ाइल को सेव करने के बाद, आपको contact पेज दिखना चाहिए और होमपेज के लिंक का अनुसरण करने में सक्षम होना चाहिए।
```

Comment:
`और होमपेज के लिंक का अनुसरण करने में सक्षम होना चाहिए।` => `और होमपेज लिंक से होमपेज पर जा सकना चाहिए।`

any thoughts on this?

Reply:
how about ```और होमपेज लिंक से होमपेज पर जा सकते है।```

Reply:
I don't think it sounds fine if we read the whole sentence, may be can you  try rephrasing the whole sentence

Reply:
ok. let us keep ```और होमपेज लिंक से होमपेज पर जा सकना चाहिए।```